client/assets/scss/main.scss

```diff
@@ -12,3 +12,17 @@
   -moz-tab-size: 4;
   tab-size: 4;
 }
+
+img {
```

Comment:
Make sure this is more specific to the images you want this height applied too. This will allow different sections of your pages to have variable images sizes.